test_endpoints.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+
+def test_post():
+    p = requests.post('http://localhost:8000/api/players/')
+    player_id = p.json()['player_id']
+    g = requests.post('http://localhost:8000/api/games/',
+                      data={'player_id': player_id})
+    assert p.status_code == 201
+    assert g.status_code == 201
+    assert len(p.json()['player_id']) == 8
+    assert len(g.json()['game_id']) == 8
+    assert g.json()['players'][0] == player_id
```

Comment:
Would be better if these were separate tests: one for creating players, one for creating games.